### PR TITLE
Ports Config Changes from Original Repo

### PR DESCRIPTION
**Pull request by Birdtalon**
Ports config changes from original repository.

I haven't touched any mapping things here as I'm not 100% familiar with them.

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -1,8 +1,8 @@
-[Feature Freeze!]: # We are currently not considering any balance or antagonist oriented pull requests. Full details, as well as ways to bypass this freeze, are available here https://github.com/tgstation/tgstation/pull/28223
+[why]: # (Please add a short description of the changes you have made in your PR. Please 
 
 
 [Changelogs]: # (Please make a changelog if you're adding, removing or changing content that'll affect players. This includes, but is not limited to, new features, sprites, sounds; balance changes; map edits and important fixes)
-[]: # (See here for how to easily make a changelog: https://github.com/tgstation/tgstation/wiki/Changelogs. An example changelog has been provided below. Please edit or remove)
+[]: # (See here for how to easily make a changelog: https://github.com/Planetfall/PlanetfallSS13/wiki/Guide-to-Changelogs An example changelog has been provided below. Please edit or remove)
```
Comment (SamHPurp) [inline .github/PULL_REQUEST_TEMPLATE.md]: URL changed with repo change.... and does this wiki page exist for us?

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 2]
```diff
@@ -1,8 +1,8 @@
-[Feature Freeze!]: # We are currently not considering any balance or antagonist oriented pull requests. Full details, as well as ways to bypass this freeze, are available here https://github.com/tgstation/tgstation/pull/28223
+[why]: # (Please add a short description of the changes you have made in your PR. Please
```
Comment (SamHPurp) [inline .github/PULL_REQUEST_TEMPLATE.md]: seems to cut off?

File: .github/CONTRIBUTING.md [hunk 3]
```diff
@@ -349,27 +327,16 @@ There is also an undocumented keyword called `static` that has the same behaviou
...
 
 * If you are proposing multiple changes, which change many different aspects of the code, you are expected to section them off into different pull requests in order to make it easier to review them and to deny/accept the changes that are deemed acceptable.
 
 * If your pull request is accepted, the code you add no longer belongs exclusively to you but to everyone; everyone is free to work on it, but you are also free to support or object to any changes being made, which will likely hold more weight, as you're the one who added the feature. It is a shame this has to be explicitly said, but there have been cases where this would've saved some trouble.
 
 * Please explain why you are submitting the pull request, and how you think your change will be beneficial to the game. Failure to do so will be grounds for rejecting the PR.
 
-## Banned content
-Do not add any of the following in a Pull Request or risk getting the PR closed:
-* National Socialist Party of Germany content, National Socialist Party of Germany related content, or National Socialist Party of Germany references
```
Comment (SamHPurp) [inline .github/CONTRIBUTING.md]: Probably keep this tbh.

File: .github/CONTRIBUTING.md [hunk 4]
```diff
@@ -1,48 +1,26 @@
...
 
-We have a [list of guides on the wiki](http://www.tgstation13.org/wiki/index.php/Guides#Development_and_Contribution_Guides) that will help you get started contributing to /tg/station with Git and Dream Maker. For beginners, it is recommended you work on small projects like bugfixes at first. If you need help learning to program in BYOND, check out this [repository of resources](http://www.byond.com/developer/articles/resources).
+We have a [list of guides on the wiki](http://www.tgstation13.org/wiki/index.php/Guides#Development_and_Contribution_Guides) that will help you get started contributing to Planetfall with Git and Dream Maker. For beginners, it is recommended you work on small projects like bugfixes at first. If you need help learning to program in BYOND, check out this [repository of resources](http://www.byond.com/developer/articles/resources).
 
-There is an open list of approachable issues for [your inspiration here](https://github.com/tgstation/-tg-station/issues?q=is%3Aopen+is%3Aissue+label%3A%22Easy+Fix%22).
+A list of open issues can be found here. (https://github.com/Planetfall/PlanetfallSS13/issues)
```
Comment (SamHPurp) [inline .github/CONTRIBUTING.md]: URLs changed with the repo change.

File: code/__DEFINES/atmospherics.dm [hunk 5]
```diff
@@ -174,3 +174,5 @@
 
 #define LAVALAND_EQUIPMENT_EFFECT_PRESSURE 50 //what pressure you have to be under to increase the effect of equipment meant for lavaland
 #define LAVALAND_DEFAULT_ATMOS "o2=14;n2=23;TEMP=300"
+
+#define PLANET_DEFAULT_ATMOS "o2=22;n2=82;TEMP=293.15"
```
Comment (SamHPurp) [inline code/__DEFINES/atmospherics.dm]: Keep this out, we'll include this in the turf/atmos PR.

File: .github/CONTRIBUTING.md (hunk 3 above)
Comment (Birdtalon) [inline .github/CONTRIBUTING.md]: I'll re-add it, but it's probably worth revisiting regardless once we get further, for now I trust our team not to make Nazi PRs.

File: code/__DEFINES/atmospherics.dm (hunk 5 above)
Comment (Birdtalon) [inline code/__DEFINES/atmospherics.dm]: Removed.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 2 above)
Comment (Birdtalon) [inline .github/PULL_REQUEST_TEMPLATE.md]: Updated. This was actually the same in the original Repo. Not sure where I was going here. A better template can be written as we go forward.

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (Birdtalon) [inline .github/PULL_REQUEST_TEMPLATE.md]: Yes I have ported that wiki page over and amended the URL now.

File: .github/CONTRIBUTING.md (hunk 4 above)
Comment (Birdtalon) [inline .github/CONTRIBUTING.md]: Redirected.

**Pull request merged**